src/Symfony/Component/RateLimiter/TokenBucketLimiter.php

```diff
@@ -38,6 +39,7 @@ public function __construct(string $id, int $maxBurst, Rate $rate, StorageInterf
         $this->rate = $rate;
         $this->storage = $storage;
         $this->lock = $lock ?? new NoLock();
+        $this->strategy = static::class;
```

Comment:
I think this state should be removed and just call `static::class` where required. Same goes for the other limiters.

Reply:
Declined in favor of key string https://github.com/symfony/symfony/pull/38257#issuecomment-699578267